tests/NuGet.Services.BasicSearchTests/TestSupport/DataInitializer.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NuGet.Services.BasicSearchTests.TestSupport
+{
+    public class DataInitializer
+    {
+        private readonly BasicSearchTestSettings _settings;
+
+        public DataInitializer(BasicSearchTestSettings settings)
+        {
+            _settings = settings;
+        }
+
+        public async Task InitializeAsync()
+        {
+            Directory.CreateDirectory(_settings.DataDirectory);
+            await WriteFileAsync("downloads.v1.json", "[]");
+            await WriteFileAsync("curatedfeeds.json", "[]");
+            await WriteFileAsync("owners.json", "[]");
+            await WriteFileAsync("rankings.v1.json", "{\"Rank\": []}");
```

Comment:
Yup, that's one option. I was also thinking that it could be set as part of the list of packages to download and include in the Lucene index (that is, the `PackageVersionIdentifier` type). If it's okay with you, I'll wait until we need it to build it.